sirepo/auth/__init__.py

```diff
@@ -462,15 +475,17 @@ def user_if_logged_in(method):
 
 
 def user_name():
+    m = cookie.unchecked_get_value(_COOKIE_METHOD)
```

Comment:
I don't understand why this is changed.

Reply:
This was just me cleaning up the code. I thought it was strange that this method wasn't prefixed with "unchecked" but could fall through and return none. Agree?

Reply:
Sometimes routines like this are not called unchecked. I think in the case of hot fixes, we should only change what's absolutely necessary. I thought the change was semantically different.

Reply:
Makes sense. Will do.

Reply:
I also see that the semantics are better. I didn't understand that it wasn't returning anything, which is bad. I think it would have blown up, because getattr would blow up so there would always be a `u`. Therefore the assert should be on _COOKIE_METHOD. It is "unchecked" because that was the only way when this was written (me thinks).